src/routes/add-place.ts

```diff
@@ -134,7 +138,6 @@ export default async function addPlace(fastify: FastifyInstance) {
     RemotePlaceCache.clear(chtApi, place.type.name);
     await RemotePlaceResolver.resolveOne(place, sessionCache, chtApi, { fuzz: true });
     place.validate();
-
```

Comment:
can you revert these white space changes?  They just add clutter to the PR

Reply:
this file `add-place.ts` should not be in this PR because you just made white space changes - please revert

Reply:
can you revert these white space changes?